src/lib/libc_preload/syscall_wrappers.c

```diff
@@ -137,12 +137,8 @@ INTERPOSE(epoll_pwait2);
 #ifdef SYS_epoll_wait // kernel entry: num=232 func=sys_epoll_wait
 INTERPOSE(epoll_wait);
 #endif
-#ifdef SYS_eventfd // kernel entry: num=284 func=sys_eventfd
-INTERPOSE(eventfd);
-#endif
-#ifdef SYS_eventfd2 // kernel entry: num=290 func=sys_eventfd2
-INTERPOSE(eventfd2);
-#endif
+// Skipping SYS_eventfd
```

Comment:
Should we instead do:

```C
#ifdef SYS_eventfd // kernel entry: num=284 func=sys_eventfd
INTERPOSE_REMAP(eventfd, eventfd2);
#endif
```

That way we still get the benefit of skipping the seccomp interception overhead whenever eventfd is called.

Reply:
Changed this to remap it instead.